.gitignore

```diff
@@ -7,3 +7,4 @@ sofa-pcm/*.d
 *.cfg
 *.pyc
 __pycache__
+.vscode
```

Comment:
Please gitignore `sofalog` as well, in case of accidentally commit.

You can revise the commit history of your own, then force-push to the same branch (notreal1995:dev).
The PR will be updated accordingly.